examples/tutorials/voice/handle-incoming-call/src/main/java/com/mycompany/app/Config.java

```diff
@@ -0,0 +1,37 @@
+package com.mycompany.app;
+
+import com.sinch.sdk.SinchClient;
+import com.sinch.sdk.domains.voice.VoiceService;
+import com.sinch.sdk.models.Configuration;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+
+@org.springframework.context.annotation.Configuration
+public class Config {
+
+  @Value("${credentials.application.api-key}")
+  String applicationApiKey;
+
+  @Value("${credentials.application.api-secret}")
+  String applicationApiSecret;
+
+  @Value("${voice.controller-path}")
+  private String controllerPath;
+
+  @Bean
+  public VoiceService voiceService() {
+
+    var configuration =
+        Configuration.builder()
+            .setApplicationKey(applicationApiKey)
```

Comment:
It feels strange to have a variable called `ApiKey` while the method mentions `Key` only.

Reply:
Right. Thank you
Refactored to use applicationKey/applicationSecret